src/js/autocomplete.js

```diff
@@ -237,13 +241,15 @@ class Autocomplete {
 			accessibleAutocomplete({
 				element: this.container,
 				id: id,
+				name: name,
```

Comment:
question: when this is `undefined` does accessible-autocomplete use it's default value instead?

Reply:
It does, defaulting to 'input-autocomplete'

https://github.com/alphagov/accessible-autocomplete#name-default-input-autocomplete